system_tests/vision.py

```diff
@@ -474,13 +474,13 @@ def tearDown(self):
             value.delete()
 
     def _assert_color(self, color):
-        self.assertIsInstance(color.red, int)
-        self.assertIsInstance(color.green, int)
-        self.assertIsInstance(color.blue, int)
+        self.assertIsInstance(color.red, (int, float))
+        self.assertIsInstance(color.green, (int, float))
+        self.assertIsInstance(color.blue, (int, float))
```

Comment:
The HTTP version return ints just an artifact of JSON encoding stripping extra data?

```node
> JSON.stringify({a: 1.0})
"{"a":1}"
```

to

```python
>>> import json
>>> json.loads('{"a": 1}')
{u'a': 1}
>>> # vs.
>>> json.loads('{"a": 1.0}')
{u'a': 1.0}
```

Should we cast them to floats?